### DISCUSSION TITLE
Tutorial / API Errors with latest version

### DISCUSSION THREAD

**Discussion by NightKnight2947** [KellanM/OpenAI-Api-Unreal#46] [General]
I downloaded the latest version because you added GPT4 Support and I wanted to know if you could make a tutorial, because I get API Errors (in the print-> error message).

In the log though it says the model 4 doesnt exist, as well as the gpt4-32k version. If I choose the gpt3.5, I get the message "You exceeded your current quota, please check your plan and billing details.".

Do I need to have a paid account?

Have a nice day.

**Comment by DeltaTwozero** [comment]
Yes, you need to setup payment to use this plugin since OpenAI charges users for API calls.
